app/labor/badge_rewarder.rb

```diff
@@ -1,5 +1,5 @@
 module BadgeRewarder
-  YEARS = { 1 => "one", 2 => "two", 3 => "three" }.freeze
+  YEARS = { 1 => "one", 2 => "two", 3 => "three", 4 => "four", 5 => "five", 6 => "six", 7 => "seven" }.freeze
```

Comment:
Decided to expand the size of this array. Ideally we probably want a better way to generate this. One possibility is [a gem like this numbers_in_words](https://github.com/markburns/numbers_in_words) gem but I figured that was outside the scope of this PR.

Reply:
This is great. I've been meaning to add four in.

This change buys us at least three years before we need to consider something new. 😋